internal/app/machined/pkg/runtime/v1alpha1/board/rpi_4/rpi_4.go

```diff
@@ -5,8 +5,8 @@
 package rpi4
 
 import (
+	_ "embed" //nolint:gci
```

Comment:
`//nolint:gci` is required by the combination of blank import, golang/lint issues (and abandonment) and gci being gci.